concourse/pipelines/create-cloudfoundry.yml

```diff
@@ -2127,6 +2220,70 @@ jobs:
                   cf set-env rubbernecker PIVOTAL_PROJECT_ID ${PIVOTAL_PROJECT_ID}
                   cf start rubbernecker
 
+        - task: deploy-paas-metrics
+          config:
+            platform: linux
+            image_resource:
+              type: docker-image
+              source:
+                repository: governmentpaas/cf-cli
+                tag: 895cf6752c8ec64af05a3a735186b90acd3db65a
+            params:
+              ENABLE_METRICS: ${ENABLE_METRICS}
```

Comment:
This ins't going to parse. You may need to modify to fit what we're doing for datadog below.